src/Evolution/DgSubcell/Tags/Jacobians.hpp

```diff
@@ -35,4 +35,76 @@ struct DetInverseJacobianLogicalToGrid : db::SimpleTag {
   static std::string name() { return "Det(InverseJacobian(Logical,Grid))"; }
   using type = std::optional<Scalar<DataVector>>;
 };
+
+/// \brief The inverse Jacobian from the element logical frame to the inertial
+/// frame at the cell centers.
+///
+/// Specifically, \f$\partial x^{\bar{i}} / \partial x^i\f$, where \f$\bar{i}\f$
+/// denotes the element logical frame and \f$i\f$ denotes the inertial frame.
+///
+/// \note stored as a `std::optional` so we can reset it when switching to DG
+/// and reduce the memory footprint (is it useful for a ComputeTag though?).
+template <size_t Dim>
+struct InverseJacobianLogicalToInertial : db::SimpleTag {
+  static std::string name() { return "InverseJacobian(Logical,Inertial)"; }
+  using type =
+      std::optional<::InverseJacobian<DataVector, Dim, Frame::ElementLogical,
+                                      Frame::Inertial>>;
+};
+
+/// Compute item for the inverse jacobian matrix from logical to
+/// inertial coordinates
+template <typename MapTagGridToInertial, size_t Dim>
+struct InverseJacobianLogicalToInertialCompute
+    : InverseJacobianLogicalToInertial<Dim>,
+      db::ComputeTag {
+  static constexpr size_t dim = Dim;
+  using base = InverseJacobianLogicalToInertial<Dim>;
+  using return_type = typename base::type;
+  using argument_tags =
+      tmpl::list<MapTagGridToInertial,
+                 evolution::dg::subcell::Tags::Coordinates<dim, Frame::Grid>,
+                 ::Tags::Time, ::domain::Tags::FunctionsOfTime,
```

Comment:
I think even on static meshes this gets reevaluated at every time step, because the value of `::Tags::Time` changes. Could be pretty bad for performance. See also #3322. Maybe just add a comment that this get reevaluated every time step even on static meshes.

Reply:
Right, we might want a time-independent and a time-dependent version in the future, if we care about performance on static meshes.